cypress.config.ts

```diff
@@ -4,12 +4,11 @@ export default defineConfig({
   // setupNodeEvents can be defined in either
   // the e2e or component configuration
   e2e: {
+    // solo necesitas agregar estas dos propiedades, no incluir este comentario
+    video: false,
+    screenshotOnRunFailure: false,
     setupNodeEvents(on, config) {
-      // modify config values examples
-      // config.defaultCommandTimeout = 10000
-
-      // IMPORTANT return the updated config object
-      return config;
+      // Codigo existente no modificar
```

Comment:
The comment on the inscruction means that on this section there was a code that should not be change, not to put that comment. Please revert the change of this section.